### ISSUE TITLE
Add class to add border to figures

### ISSUE THREAD

**Issue by beep** [OpenNews/opennews-source#72]
On https://opennews-source-staging.herokuapp.com/articles/how-npr-transcribes-and-fact-checks-debates-live/, and a few other articles, the lead image feels a little float-y on a white background:

![screen shot 2017-02-02 at 11 43 48 am](https://cloud.githubusercontent.com/assets/124493/22558779/f2228750-e93c-11e6-83c0-4d8244cd36e7.png)

Should we default to a thin gray border on those images? Or at least give editors the option to add one?

/ping #22 #71

**Comment by kissane** [comment]
I would love the option to add one with a class, please—I don't think we need one by default, though. Thank you!

**Comment by beep** [comment]
@kissane On it!

**Comment by beep** [comment]
@ryanpitts If this is the markup for the lead figure:

```
<div class="figure figure-article-lead">
    <figure>
        <img src="/tmpl/img/fpo/article-lead.jpg" alt="" />
        <figcaption>
            <p>Our printed ballot guide for voters to take to the polls—a product of the testing described in this article.</p>
        </figcaption>
    </figure>
</div><!-- /end .figure.figure-article-minor -->
```

Where’s the best place for a toggle class? On the image, or on the outer `div`, or somewhere else?

Asking as I realized I’m not sure what the author experience looks like; are authors entering in HTML, or is there some sort of checkbox being toggled for `add a border`? (cc @kissane)

**Comment by ryanpitts** [comment]
in this case, I would need to add a checkbox next to the `article.image` field in the admin, so I would say implement wherever makes the best sense in the HTML and I'll incorporate a simple `if` check in the template.

**Comment by beep** [comment]
@ryanpitts perfect, thanks! two shakes.

**Comment by beep** [comment]
@ryanpitts okay so after wasting your time with a dumb question i gave you two options https://github.com/OpenNews/opennews-source/commit/12b6635484f5ecea887bc1eb22802b908230d46b

1. Add `.has-border` to a `div.figure` _or_
2. Add `.has-border` directly to an image.

I’ll leave it to you which you prefer for the lead figure—[I went with the former option](https://github.com/OpenNews/opennews-source/commit/12b6635484f5ecea887bc1eb22802b908230d46b#diff-2100d19a9b8aed560d98e208e977a466R93)—but I figured both options would be handy for anyone dropping HTML directly into articles.
